scripts/install

```diff
@@ -296,34 +297,42 @@ systemctl disable hostapd
 systemctl stop hostapd
 systemctl unmask hostapd # On some system e.g. RPi, for some reason hostapd is masked after install ...
 
-#=================================================
-# RELOAD NGINX AND PHP-FPM
-#=================================================
-ynh_print_info "Reloading nginx web server..."
-
-systemctl restart php7.0-fpm
-systemctl reload nginx
-
 # Remove IPv6 address set if there is a VPN installed
 if [[ $ip6_addr != none ]]; then
   if ip -6 address show dev tun0 2> /dev/null | grep -q "${ip6_addr}/"; then
     ip address delete "${ip6_addr}/128" dev tun0 &> /dev/null
   fi
 fi
 
-# register the service
-yunohost service add $service_name --description "Creates a Wi-Fi access point" --need_lock
+#=================================================
+# INTEGRATE SERVICE IN YUNOHOST
+#=================================================
+ynh_script_progression --message="Integrating service in YunoHost..."
 
-# enable and start the service if device is present
+yunohost service add $service_name --description "Creates a Wi-Fi access point" --log_type "systemd" --need_lock
```

Comment:
Zblerg, not sure why adding --log_type=systemd ... in fact it's kind of a deprecated option nowadays :/

```suggestion
yunohost service add $service_name --description "Creates a Wi-Fi access point" --need_lock
```

Reply:
adding `--log_type=systemd` allows to directly have journalctl informations in the service tab. For example for pleroma:

https://github.com/YunoHost-Apps/pleroma_ynh/blob/e8a4a37c2fa35255d0ca3927df8da09e3c5829b6/scripts/install#L256

![image](https://user-images.githubusercontent.com/30271971/85922520-4a4d5f80-b884-11ea-8585-a8806c65584a.png)

Reply:
This was indeed necessary in the past, but now yunohost will return the journalctl logs by default, no need to specify that option (or if you do need it to work, it should be considered as a bug)

Reply:
thanks, I didn't knew that. If you can merge https://github.com/labriqueinternet/hotspot_ynh/pull/65 to solve that. I don't have rights on https://github.com/labriqueinternet

Reply:
@yalh76 it's merged